<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Ingresar</string>
       </attribute>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>20</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Procesos registrados</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="plainTextEdit_procesos">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>50</y>
          <width>311</width>
          <height>441</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>401</width>
          <height>211</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="2" column="2">
          <widget class="QComboBox" name="comboBox">
           <property name="currentText">
            <string/>
           </property>
          </widget>
         </item>
         <item row="4" column="1" colspan="3">
          <widget class="QLineEdit" name="lineEdit_num"/>
         </item>
         <item row="2" column="3">
          <widget class="QSpinBox" name="spinBox"/>
         </item>
         <item row="1" column="1" colspan="3">
          <widget class="QLineEdit" name="lineEdit_nombre"/>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="spinBox_2"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Tiempo max estimado</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Nombre programador</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="3">
          <widget class="QSpinBox" name="spinBox_tiempo">
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Operación a realizar</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="4">
          <widget class="QPushButton" name="pushButton_registrar">
           <property name="text">
            <string>Registrar</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Número de programa</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="4">
          <widget class="QLabel" name="label_7">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>210</height>
            </size>
           </property>
           <property name="text">
            <string>Informacion del proceso</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Simulación</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>46</y>
          <width>431</width>
          <height>463</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="8" column="2">
          <widget class="QLCDNumber" name="lcdNumber_transcurrido">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="digitCount">
            <number>3</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
          </widget>
         </item>
         <item row="8" column="0" rowspan="2">
          <widget class="QPlainTextEdit" name="plainTextEdit_proejecucion"/>
         </item>
         <item row="8" column="1">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Tiempo transcurrido en ejecucion</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="9" column="2">
          <widget class="QLCDNumber" name="lcdNumber_restante">
           <property name="digitCount">
            <number>3</number>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Tiempo restante a ejecutar</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="3">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Lote en ejecución</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="3">
          <widget class="QTableWidget" name="tableWidget_lote"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>453</x>
          <y>46</y>
          <width>311</width>
          <height>463</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Procesos terminados</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="tableWidget_terminados"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>751</width>
          <height>41</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Tiempo Global</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdNumber_global">
           <property name="digitCount">
            <number>3</number>
           </property>
           <property name="intValue" stdset="0">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>            </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_pendientes">
           <property name="text">
            <string>Lotes pendientes : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_procesos">
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_iniciar">
           <property name="text">
            <string>Iniciar la simulación</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>803</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
